lib/github/ldap.rb

```diff
@@ -98,6 +105,9 @@ def initialize(options = {})
       # configure both the membership validator and the member search strategies
       configure_search_strategy(options[:search_strategy])
 
+      # configure both the membership validator and the member search strategies
+      configure_user_search_strategy(options[:global_catalog_user_search])
```

Comment:
I don't think the `user_search_strategy` configuration key should be named `global_catalog` since that's a specific strategy. I know we only have one option here, but I'd prefer we pass in the strategy as a key (like we do with `search_strategy` above).